### ISSUE TITLE
Add configuration to not count SETs towards log threshold

### ISSUE THREAD

**Issue by Hayden-IO** [sigstore/sigstore-go#54]
**Description**

Discussed more in https://github.com/sigstore/sigstore-go/pull/51 and https://github.com/sigstore/sigstore-go/issues/44. This should be off by default for v0.2+ bundles, since log inclusion proofs are required, and on by default for v0.1 bundles, since proofs are optional.

Blocked by #51 being merged

**Comment by Hayden-IO** [comment]
Going to close as won't fix - Thinking more on this, calling SETs promises is a bit inaccurate, since it's a signed commitment from the log and contains a log index, meaning it commits to the same state as an inclusion proof + a signed checkpoint. I'll open a new issue when we integrate co-signed checkpoints to configure a threshold for the number of those, which will require an inclusion proof.
